### PR TITLE
Port testRASAPI into openj9

### PR DESCRIPTION
**Pull request by OscarQQ**
related: backlog issue 426
[ci skip]

Signed-off-by: OscarQQ <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
| Build       | Platform           | Java Version  | JDK Impl |
|:-------------:|:-------------:|:-----:|:-----------:|
|[Grinder/5727](https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5727/)|ppc64_aix,ppc64le_linux,s390x_linux,x86-64_linux,x86-64_mac,x86-64_windows|8,11|OpenJ9|

File: test/functional/RasapiTest/build.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	<property name="source.dir" location="./src" />
+	<property name="dummy.source.dir" location="./dummy_src" />
+	<property name="build.dir" location="./build" />
+	<property name="dist.dir" location="./dist" />
+	<property name="src" location="." />
+	<property name="LIB" value="junit4"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+	
+	<property name="dir.bootcompiler" value="/bluebird/tools/boot-compiler"/>
+	<property name="build.compiler.java" value="${dir.bootcompiler}/sdk70/latest/bin/javac" />
```
Comment (llxia) [inline test/functional/RasapiTest/build.xml:39]: `build.compiler.java` is not used. Please remove `build.compiler.java`  and `dir.bootcompiler`

File: test/functional/RasapiTest/playlist.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		<!-- temporarily disable this test on osx; backlog issue 426 -->
+		<platformRequirements>^os.osx</platformRequirements>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<impls>
+			<impl>openj9</impl>
```
Comment (llxia) [inline test/functional/RasapiTest/playlist.xml:40]: Why we exclude this test for ibm?

Comment (OscarQQ) [reply test/functional/RasapiTest/playlist.xml:42]: In order to disbale the tests on vmfarm, I just tested it on vmfarm and the results are very inconsistent e.x. job_output.php?id=666639064

File: test/functional/RasapiTest/build.xml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    	</copy>
+	</target>
+	
+    <target name="build" depends="compile">
+    	<mkdir dir="${DEST}"/>
+		<copy todir="${DEST}">
+			<fileset dir="${dist.dir}" includes="*.xml,**/*.jar,**/*.xml"/>
+			<fileset dir="${src}" includes="*.mk"/>
+		</copy>
+    </target>
```
Comment (llxia) [inline test/functional/RasapiTest/build.xml:109]: This test is for openj9/ibm, so we should disable the compilation on the other jvm as well.

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by llxia** [comment]
jenkins test extended osx jdk8,jdk11

**Comment by llxia** [comment]
The test passed on `ub16p8j92`: https://ci.eclipse.org/openj9/job/Grinder/1424/


And it is failed on `ub16-ppcle-2`: https://ci.eclipse.org/openj9/job/Grinder/1425/console
```
00:10:49.868  /home/jenkins/jenkins-agent/workspace/Grinder/jvmtest/functional/RasapiTest/test.xml:65: taskdef class org.apache.tools.ant.taskdefs.optional.junit.JUnitTask cannot be found
00:10:49.868   using the classloader AntClassLoader[/home/jenkins/jenkins-agent/workspace/Grinder/jvmtest/functional/RasapiTest/junit4.jar]
00:10:49.868  
00:10:49.868  Total time: 0 seconds
00:10:49.868  
00:10:49.868  testRASAPI_0_FAILED
```

From [here](https://stackoverflow.com/questions/6568634/how-to-solve-cause-the-class-org-apache-tools-ant-taskdefs-optional-junit-juni/26719213#26719213), the error is due to ant-junit.jar is missing. 

@AdamBrousseau It looks like the above two ppcle machines are configured differently. Could we update `ub16-ppcle-2`? Thanks.

**Comment by AdamBrousseau** [comment]
Updated the 3 plinux at OSU. These were set up long ago so they were on ant 1.9. Updated to 1.10 and ant-contrib 1.0b3. Please try again.

**Comment by llxia** [comment]
Thanks a lot @AdamBrousseau . 
Reran on ub16-ppcle-2: https://ci.eclipse.org/openj9/job/Grinder/1428/

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Review by llxia — APPROVED** [review]
LGTM

**Comment by fjeremic** [comment]
> The first line should be less than 70 characters, where reasonable, and should be written in sentence case **(capitalize the first letter)** preferably not ending in a period.

https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md#commit-guidelines

I guess we already ran the testing on this one. Up to the committer to choose whether to merge or not. Just noting this for your future PRs.

**Pull request merged**